AdminUi/src/AdminUi/ClientApp/src/app/services/identity-service/identity.service.ts

```diff
@@ -5,6 +5,7 @@ import { HttpResponseEnvelope } from "src/app/utils/http-response-envelope";
 import { PagedHttpResponseEnvelope } from "src/app/utils/paged-http-response-envelope";
 import { environment } from "src/environments/environment";
 import { Quota } from "../quotas-service/quotas.service";
+import { Tier } from "../tier-service/tier.service";
```

Comment:
Could probably avoid this reference by just passing `tier.id` to the `updateIdentityTier` method.